.github/workflows/stale-issues.yml

```
@@ -0,0 +1,23 @@
+name: Send comment on stale issues
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'ScratchAddons'
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'Ping! There has been no activity for 7 days.'
+          stale-pr-message: 'Ping! There has been no activity for 7 days.'
+          only-label: 'status: awaiting answer/followup'
+          days-before-stale: 7
+          days-before-close: -1
+          stale-issue-label: ''
+          stale-pr-label: ''
+          operations-per-run: 5
+    permissions:
+      issues: write
+      pull-requests: write
```

Super nitpick: Would it be possible to use the bot for this?

```suggestion
    runs-on: ubuntu-latest
    if: github.repository_owner == 'ScratchAddons'
    steps:
      - name: Generate token
        uses: tibdex/github-app-token@v1
        id: generate-token
        with:
          app_id: ${{ secrets.BOT_APP_ID }}
          private_key: ${{ secrets.BOT_PRIVATE_KEY }}
      - name: Execute Action
        uses: actions/stale@v6
        with:
          stale-issue-message: 'Ping! There has been no activity for 7 days.'
          stale-pr-message: 'Ping! There has been no activity for 7 days.'
          only-label: 'status: awaiting answer/followup'
          days-before-stale: 7
          days-before-close: -1
          stale-issue-label: ''
          stale-pr-label: ''
          operations-per-run: 5
          repo-token: ${{ steps.generate-token.outputs.token }}
```